reminder/module.py

```diff
@@ -0,0 +1,409 @@
+            "Reminder ID {id} succesfully sent to {user}".format(
+                id=item.idx, user=reminded_user.display_name
+            ),
+        )
+
+    async def _process_text(
+        self, ctx: commands.Context, datetime_str: str, text: Optional[str]
+    ):
+        if text is not None:
+            lines = ctx.message.content
+
+            lines = lines.split(datetime_str)[1]
+
+            lines = re.split(" ", lines)
+            while "" in lines:
+                lines.remove("")
+            text = " ".join(lines)
+
+            if len(text) > 1024:
+                text = text[:1024]
+                text = text[:-3] + "```" if lines.count("```") % 2 != 0 else lines
+
+        return text
+
+    async def _get_embed(self, utx, query):
+        reminder_user = await self._get_member(query.recipient_id, query.guild_id)
+
+        if reminder_user is None:
+            reminder_user_name = "_(Unknown user)_"
+        else:
+            reminder_user_name = nextcord.utils.escape_markdown(
+                reminder_user.display_name
+            )
+
+        embed = utils.discord.create_embed(
+            author=reminder_user,
+            title=_(utx, "Reminder"),
+        )
+
+        if query.author_id != query.recipient_id:
+            embed.add_field(
+                name=_(utx, "Reminded by"),
+                value=reminder_user_name,
+                inline=True,
+            )
+        if query.message != "":
+            embed.add_field(
+                name=_(utx, "Message"),
+                value=query.message,
+                inline=False,
+            )
+        embed.add_field(name=_(utx, "URL"), value=query.permalink, inline=True)
+
+        return embed
+
+    async def _send_reminder_list(self, ctx, query):
+        reminders = []
+
+        for item in query:
+            author = await self._get_member(item.author_id, item.guild_id)
+            remind = await self._get_member(item.recipient_id, item.guild_id)
+
+            author_name = author.display_name if author is not None else "(unknown)"
+            remind_name = remind.display_name if remind is not None else "(unknown)"
+
+            reminder = ReminderDummy()
+            reminder.idx = item.idx
+            reminder.author_name = author_name
+            reminder.remind_name = remind_name
+            reminder.remind_date = item.remind_date
+            reminder.status = item.status.name
+            reminder.url = item.permalink.replace("https://discord.com/channels/", "")
+
+            reminders.append(reminder)
+
+        table_pages: List[str] = utils.text.create_table(
+            reminders,
+            {
+                "idx": _(ctx, "Reminder ID"),
+                "author_name": _(ctx, "Author"),
+                "remind_name": _(ctx, "Reminded"),
+                "remind_date": _(ctx, "Remind date"),
+                "status": _(ctx, "Status"),
+                "url": "https://discord.com/channels/",
```

Comment:
That URL header doesn't look pretty, I'd use "URL stub" or something similar.

Reply:
Fixed